openedx/core/djangoapps/user_authn/views/tests/test_login.py

```diff
@@ -1,8 +1,11 @@
+#coding:utf-8
```

Comment:
It's 2018, people. Everything should assume utf8 by default. smh

Reply:
It's Python 2.7.... :(  Once we get to 3.x, we don't need them any more.